Sources/AsyncHTTPClient/AsyncAwait/HTTPClientRequest.swift

```diff
@@ -14,10 +14,18 @@
 
 import NIOCore
 import NIOHTTP1
+#if compiler(>=5.7)
```

Comment:
We have just drop 5.6, we can remove this now.

Reply:
cool, will remove